articles/introduction-to-syncing.mdx

```diff
@@ -1,16 +1,52 @@
 ---
 _uuid: "c3502cz4-d666-4fa7-b340-c27333c9e7ef"
-_created_at: "2021-09-15 23:13:40 +0000"
-title: "Introduction to syncing"
+_created_at: "2022-03-14 00:00:00 +0000"
+title: "Introduction to Syncing"
 nav_title: "Introduction"
 published: true
 image: "ye_olde_images/i18n-enable-i18n.png"
-description: "Make your website more accessible to an international audience."
+description: "Your source files are the most important part of any site. Learn how GIT works on CloudCannon."
```

Comment:
For consistency with the rest of the docs (and Git itself) here (and the rest of this page) should be `Git` rather than `GIT`

Reply:
```suggestion
description: "Your source files are the most important part of any site. Learn how Git works on CloudCannon."
```